src/main/java/com/jcabi/http/request/BaseRequest.java

```diff
@@ -453,12 +458,89 @@ public RequestURI port(final int num) {
     }
 
     /**
-     * Base URI.
+     * Body of a request with a form that has attachments.
+     * @todo #87:1h Implement and unit test method formParam(String, Object)
```

Comment:
@amihaiemil I would also add todo for refactoring this class to get rid of "GodClass" and possibly the TooManyMethods (second one might not be possible, but maybe it is using and old qulice that was counting also inner class methods)